constraintFunctions.f90

```diff
@@ -149,6 +149,9 @@ subroutine getEnvVarsAtT( t )
     character(len=maxEnvVarNameLength) :: this_env_var_name, orderedEnvVarNames(size( envVarNames ))
     logical :: pressure_set, rh_set, temp_set
 
+    ! TODO, is this the best place to call calcTheta?
```

Comment:
I've not done a full review of it, but it's generally in a good enough state - I don't think there's any need to improve it.